### PR TITLE
Some description in vagueish tags and an alias for `openjdk` -> `jdk`

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

File: tags/guide/code.ytag [hunk 1]
```diff
@@ -2,6 +2,9 @@ type: text
 
 ---
 
+You can use codeblocks in discord as shown below:
 \`\`\`java
 code
 \`\`\`
+
+You can also specify the syntax highlighting to use by specifying the language after the last `
```
Comment (user11681) [inline tags/guide/code.ytag:10]: Maybe mention that the last backtick is at the top?

**Review by user11681 — APPROVED** [review]

File: tags/link/asmr.ytag [hunk 2]
```diff
@@ -2,5 +2,8 @@ type: text
 
 ---
 
+asmr is a concept for a future replacement for Mixin.
```
Comment (YTG1234) [inline tags/link/asmr.ytag:5]:
```suggestion
ASM-Regex (ASMR) is a concept for a future replacement for Mixin.
```

**Comment by TehcJS** [comment]
hi hello yes

i am tehc

i have reviewed your pr and :iapprove:

**Review by TehcJS — APPROVED** [review]
yes

**Review by YTG1234 — APPROVED** [review]
I am approving this PR.

**Review by TehcJS — APPROVED** [review]
oh you don't have to approve it twice

**Review by user11681 — APPROVED** [review]

**Pull request merged**